hw/bsp/native-armv7/include/bsp/bsp.h

```diff
@@ -16,24 +16,32 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-#include <stdio.h>
-#include <string.h>
-#include <unistd.h>
-
-#include "os/os.h"
-#include "os_priv.h"
-
-void
-__assert_func(const char *file, int line, const char *func, const char *e)
-{
-    char msg[256];
-
-    if (file) {
-        snprintf(msg, sizeof(msg), "assert at %s:%d\n", file, line);
-    } else {
-        snprintf(msg, sizeof(msg), "assert @ %p\n",
-                 __builtin_return_address(0));
-    }
-    write(1, msg, strlen(msg));
-    _exit(1);
+
+#ifndef __NATIVE_ARMV7_BSP_H
+#define __NATIVE_ARMV7_BSP_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Define special stackos sections */
+#define sec_data_core
+#define sec_bss_core
+#define sec_bss_nz_core
+
+/* More convenient section placement macros. */
+#define bssnz_t
+
+/* LED pins */
+#define LED_BLINK_PIN   (0x1)
+
+/* UART info */
+#define CONSOLE_UART    	"uart0"
```

Comment:
Dont' forget to remove this line, and the other one in native-mips bsp when pull request #314 gets merged.